src/integration_tests.zig

```diff
@@ -523,184 +523,6 @@ test "in-place upgrade" {
-            }
-
-            for (0..ticks_max) |tick| {
-                std.time.sleep(2 * std.time.ns_per_s);
-
-                for (0..replica_count) |replica_index| {
-                    if (tick == upgrade_tick[replica_index]) {
-                        assert(!context.replica_upgraded[replica_index]);
-                        try context.upgrade_replica(replica_index);
-                        assert(context.replica_upgraded[replica_index]);
-                    }
-                }
-
-                const replica_index = context.prng.index(context.replicas);
-                const crash = context.prng.chance(ratio(1, 4));
-                const restart = context.prng.chance(ratio(1, 2));
-
-                if (context.replicas[replica_index] == null and restart) {
-                    try context.spawn_replica(replica_index);
-                } else if (context.replicas[replica_index] != null and crash) {
-                    try context.kill_replica(replica_index);
-                }
-            }
-
-            for (0..replica_count) |replica_index| {
-                assert(context.replica_upgraded[replica_index]);
-                if (context.replicas[replica_index] == null) {
-                    try context.spawn_replica(replica_index);
-                }
-            }
-
-            workload_thread.join();
-            assert(workload_exit_ok);
-        }
-
-        fn install_replica(
-            context: *Context,
-            destination: []const u8,
-            version: enum { past, current },
-        ) !void {
-            try context.shell.cwd.copyFile(
-                switch (version) {
-                    .past => tigerbeetle_past,
-                    .current => tigerbeetle,
-                },
-                context.shell.cwd,
-                destination,
-                .{},
-            );
-            try context.shell.file_make_executable(destination);
-        }
-
-        fn spawn_replica(context: *Context, replica_index: usize) !void {
-            assert(context.replicas[replica_index] == null);
-            context.replicas[replica_index] = try context.shell.spawn(.{},
-                \\{tigerbeetle} start --addresses={addresses} {datafile}
-            , .{
-                .tigerbeetle = tigerbeetle,
-                .addresses = addresses,
-                .datafile = context.replica_datafile[replica_index],
-            });
-        }
-
-        fn kill_replica(context: *Context, replica_index: usize) !void {
-            assert(context.replicas[replica_index] != null);
-            _ = context.replicas[replica_index].?.kill() catch {};
-            context.replicas[replica_index] = null;
-        }
-    };
-
-    const seed = std.crypto.random.int(u64);
-    log.info("seed = {}", .{seed});
-    var context = try Context.init(.{ .seed = seed });
-    defer context.deinit();
-
-    try context.run();
-}
-
```

Comment:
Was this lost by accident?

Reply:
It was committed by accident, as it was nowhere near done!